### DISCUSSION TITLE
Liquidsoap 2.3.x compatibility

### DISCUSSION THREAD

**Discussion by iSerganov** [Moonbase59/autocue#43] [Q&A]
Hey there!

First of all I would like to say thank you to developers for the great job - `autocue` is a super cool tool!

I noticed README states that `autocue` is not compatible with the latest 2.3x Liquidsoap release.
Also LS the release notes say the following:

```
Autocue, the method for computing the perfect crossfade has also been polished in this release. We have now switched to our internal implementation, which is available when enabling our ffmpeg support.

You should be able to enable it by adding this to your script:

enable_autocue_metadata()

With this, all requests should resolve with the appropriate metadata so that any crossfade produces perfect transitions!
```

Does that mean that `autocue` is no longer needed and the same effect (fade-in/out, loudness, cue points etc. pre-calculations and auto corrections) can be achieved with internal Liquidsoap tooling only?

**Comment by LordHelmchen666** [comment]
These are two different autocue solutions. Both are giving nearly the same results. `autocue.internal` is now integrated in LS 2.3 and it always calculating on-the-fly. This one `autocue.cue_file` can pre-tag your files. Which is good if you have long files for example prerecorded Shows.
cue_file also works with LS 2.3 with sligthly modifications.
